From 32b4b14d6b79102c86520a032e5253b686c17867 Mon Sep 17 00:00:00 2001
From: Wenbin Song <wenbin.song@nxp.com>
Date: Thu, 16 Jun 2022 13:10:52 +0800
Subject: [PATCH 2/2] SOEM: Remalloc netc buf to non-cached partition

Fix the soem_gpio_pulse example don't work for DDR release.

Signed-off-by: Wenbin Song <wenbin.song@nxp.com>
---
 evkmimx8mm/demo_apps/soem_gpio_pulse/soem_gpio_pulse.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/evkmimx8mm/demo_apps/soem_gpio_pulse/soem_gpio_pulse.c b/evkmimx8mm/demo_apps/soem_gpio_pulse/soem_gpio_pulse.c
index fa67da4e2..ffcbe8748 100644
--- a/evkmimx8mm/demo_apps/soem_gpio_pulse/soem_gpio_pulse.c
+++ b/evkmimx8mm/demo_apps/soem_gpio_pulse/soem_gpio_pulse.c
@@ -157,9 +157,10 @@ AT_NONCACHEABLE_SECTION_ALIGN(enet_tx_bd_struct_t g_txBuffDescrip[ENET_TXBD_NUM]
  * If use cacheable region, the alignment size should be the maximum size of "CACHE LINE SIZE" and "ENET_BUFF_ALIGNMENT"
  * If use non-cache region, the alignment size is the "ENET_BUFF_ALIGNMENT".
  */
-SDK_ALIGN(uint8_t g_rxDataBuff[ENET_RXBD_NUM][SDK_SIZEALIGN(ENET_RXBUFF_SIZE, ENET_BUFF_ALIGNMENT)],
+
+AT_NONCACHEABLE_SECTION_ALIGN(uint8_t g_rxDataBuff[ENET_RXBD_NUM][SDK_SIZEALIGN(ENET_RXBUFF_SIZE, ENET_BUFF_ALIGNMENT)],
           ENET_BUFF_ALIGNMENT);
-SDK_ALIGN(uint8_t g_txDataBuff[ENET_TXBD_NUM][SDK_SIZEALIGN(ENET_TXBUFF_SIZE, ENET_BUFF_ALIGNMENT)],
+AT_NONCACHEABLE_SECTION_ALIGN(uint8_t g_txDataBuff[ENET_TXBD_NUM][SDK_SIZEALIGN(ENET_TXBUFF_SIZE, ENET_BUFF_ALIGNMENT)],
           ENET_BUFF_ALIGNMENT);
 
 enet_buffer_config_t buffConfig[] = {{
-- 
2.17.1

